microsoft-edge/devtools-guide-chromium/memory-problems/memory-101.md

```diff
@@ -192,46 +173,47 @@ A typical JavaScript object can be one of two array types:
 
 When there are a small number of properties, the properties are stored internally in the JavaScript object.
 
-**Map** is an object that describes both the kind of object it is and the layout.  For example, maps are used to describe implicit object hierarchies for [fast property access](https://v8.dev/blog/fast-properties).
-
-### Object groups
+**Map** is an object that describes both the kind of object it is and the layout. For example, maps are used to describe implicit object hierarchies for [fast property access](https://v8.dev/blog/fast-properties).
 
-Each _native objects group_ is made up of objects that hold mutual references to each other.  Consider, for example, a DOM subtree where every node has a link to the relative parent and links to the next child and next sibling, thus forming a connected graph.
+<!-- ------------------------------ -->
+#### Object groups
+<!-- This doesn't seem very actionable to users. We describe groups, but don't say where they appear in the tool and what's interesting about them. We then talk about native objects, seemingly out of the blue. -->
```

Comment:
@sethbrenith See my HTML comment here. Are object groups still a thing in V8, and do they play a role in memory consumption that web developers should know about?

Reply:
I don't think any of this is relevant anymore; let's get rid of the section.